config.yaml

```diff
@@ -1,3 +1,52 @@
 # Copyright 2023 Canonical Ltd.
 # See LICENSE file for licensing details.
-options: {}
+options:
+  webserver_workers:
+    type: int
+    description: The number of webserver worker processes for handling requests.
+  webserver_threads:
+    type: int
+    description: >
+      The number of worker threads for handling requests. Run each webserver worker with the specified number of threads.
```

Comment:
Perhaps this is redundant? "Run each webserver worker with the specified number of threads."

Reply:
Updated, thanks!